src/wp-admin/includes/upgrade.php

```diff
@@ -878,6 +878,10 @@ function upgrade_all() {
 		upgrade_560();
 	}
 
+	if ( $wp_current_db_version < 51769 ) {
+		upgrade_570();
```

Comment:
In this form, it's a one-time only upgrade. Should we make it possible to retry at a later time (e.g. after migrating the site to newer MySQL)? If yes, what would be the best approach to do that?